biohansel/subtype/qc/checks.py

```diff
@@ -16,8 +16,8 @@ def is_overall_coverage_low(st: Subtype, df: pd.DataFrame, p: SubtypingParams) -
             or not st.is_fastq_input():
         return None, None
 
```

Comment:
not sure if having multiple return statements is appropriate? Perhaps, just have two variables that represent your return values, and just return those two variables right at the end of the function?

Reply:
It's a matter of preference and what you feel comfortable doing. Multiple returns were used here to avoid lots of nesting (similar to using `break` or `continue` in a loop). Simple conditions are checked first to give an easy out before trying to do any more in-depth checks.